src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -191,12 +191,12 @@ def getElementsStatusAndJobsByWorkflow(self, inboxFlag=False, stale=True):
         options = {'reduce' : True, 'group_level': 2}
         if stale:
             options['stale'] = 'update_after'
-        data = db.loadView('WorkQueue', 'elementsDetailByWorkflowAndStatus',
+        data = db.loadView('WorkQueue', 'jobStatusByRequest',
                                 {'reduce' : True, 'group_level': 2})
```

Comment:
I have this line fixed in my PR 7519, it should be `options` instead of this dict of options.